src/JuMP.jl

```diff
@@ -386,15 +386,16 @@ getupperbound(v::Variable) = v.m.colUpper[v.col]
 function setvalue(v::Variable, val::Number)
     v.m.colVal[v.col] = val
     if v.m.colCat[v.col] == :Fixed
-        v.m.colLower[v.col] = val
-        v.m.colUpper[v.col] = val
+        error("setvalue use for setting bounds of fixed variables was deprecated in favour of JuMP.fix(v::Variable, val::Number), which changes bounds of fixed variables and fixes previously not fixed variables ")
```

Comment:
Deprecated would mean that we still support it but throw a warning. This isn't the case. ``s/favour/favor/``